docs/src/layouts/MainLayout.vue

```diff
@@ -1,74 +1,80 @@
+      <q-toolbar class="q-pa-md row justify-between">
         <q-btn flat @click="showDrawer = !showDrawer" round dense icon="menu" v-if="$q.screen.xs" color="lp-primary" />
-        <q-toolbar-title :class="$q.screen.xs? 'row justify-center items-center':''" class="add-vertical-bar position-relative">
-          <q-avatar>
-            <q-icon name="img:homepage-icons/quasar-logo-blue.svg" size="lg"/>
-          </q-avatar>
-          QUASAR
-        </q-toolbar-title>
-        <q-space v-if="$q.screen.gt.xs"/>
+        <div :class="$q.screen.xs? 'row justify-center items-center':''" class="add-vertical-bar position-relative">
+          <picture class="quasar-logo cursor-pointer" @click="$router.push({name: 'home'})">
+            <source srcset="https://github.com/quasarframework/quasar-art/blob/master/dist/svg/logo-dark.svg?raw=true" media="(min-width:810px) and (max-width:1080px)" width="48" height="48">
+            <img src="https://github.com/quasarframework/quasar-art/blob/master/dist/svg/logo-horizontal-dark.svg?raw=true" width="236" height="48" >
+          </picture>
+        </div>
 
-        <div class="row items-center">
+        <div class="row items-center text-size-16">
           <div v-if="$q.screen.gt.xs" class="toolbar-menu-items">
             <q-btn
               label="Docs"
-              color="white"
-              flat
-              :to="{ name: 'lpDocs' }"
-            />
-            <q-btn
-              label="Features"
-              color="lp-light"
+              :color="$route.name === 'lpDocs'? 'white' : 'lp-light'"
               flat
-              class="q-py-sm q-px-md"
+              :to="{ name: 'docPages' }"
+              size="16px"
             />
+<!--            <q-btn-->
+<!--              label="Features"-->
+<!--              :color="$route.name === 'features'? 'white' : 'lp-light'"-->
+<!--              flat-->
+<!--              class="q-py-sm q-px-md"-->
+<!--              size="16px"-->
+<!--            />-->
+<!--            <q-btn-->
+<!--              :color="$route.name === 'services'? 'white' : 'lp-light'"-->
+<!--              flat-->
+<!--              label="Services"-->
+<!--              size="16px"-->
+<!--            />-->
             <q-btn
-              color="lp-light"
-              flat
-              label="Services"
-            />
-            <q-btn
-              color="lp-light"
+              :color="$route.name === 'lpComponents'? 'white' : 'lp-light'"
               flat
               label="Components"
               class="q-py-sm q-px-md"
               :to="{ name: 'lpComponents' }"
+              size="16px"
             />
             <q-btn
-              color="lp-light"
+              :color="$route.name === 'sponsors'? 'white' : 'lp-light'"
               flat
               label="Become sponsor"
               class="q-py-sm q-px-md"
+              size="16px"
+              to="/sponsors-and-backers"
```

Comment:
Try being consistent at least in the code spatial proximity, either using always the path or the route name
I prefer the latter, but it's usually not worth the trouble for a few links

Reply:
The inconsistency was due to the fact that most of the new routes I added were auto generated unlike the previous which I had personally defined in routes.js. I have gone with using just links